suite-native/module-passphrase/src/components/PassphraseScreenHeader.tsx

```diff
@@ -44,8 +75,23 @@ export const PassphraseScreenHeader = () => {
                 colorScheme="tertiaryElevation1"
                 accessibilityRole="button"
                 accessibilityLabel="close"
-                onPress={handleClose}
+                onPress={handlePress}
             />
+            <BottomSheet isVisible={shouldShowWarningBottomSheet} onClose={handleCloseBottomSheet}>
+                <VStack justifyContent="center" alignItems="center" padding="small" spacing="large">
```

Comment:
No reason to. Let's prefer styling by props when the component was designed this way.

Reply:
Hmm, I guess this is personal preference. To me it seems better. We agreed long time ago to use inline props when possible, with exception when style is applied, then it makes sense to merge them I guess. We can open this issue again if you’d like, but let’s not waste time on code reviews on this please since there is no agreement on this >4 props and the whole code base is not following anything similar…

Reply:
I was told, that there is a limit of 3 informally agreed on. And as you said, personally I prefer to have the style in such case. But no extra strong opinion on this. Lets close this discussion

Reply:
I'm happy to discuss this, no problem. If it was said that 3 is max, I must have forgotten 😄 If so, we can do it that way but IMO both are legit in this case and it shouldn't really matter